docs/samples/calling/app.js

```diff
@@ -161,6 +162,54 @@ function changeEnv() {
   enableProduction.innerHTML = enableProd ? 'In Production' : 'In Integration';
 }
 
+const guestUrl = 'https://webexapis.com/v1/guests/token';
+const guestIssuerAccessToken = 'ODVkZGEyNzItMzMxMy00Y2ExLWJlMTktMjYzZGFjZWU5ZjE4NGEwN2Q3Y2ItNGM2_P0A1_881af7bb-8586-4b10-9ded-43d95f715190'; // Validity: 14 days
+
+// Guest access token via Service App
+async function getGuestAccessToken() {
+  const response = await fetch(guestUrl, {
+    method: 'post',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${guestIssuerAccessToken}`,
+    },
+    body: JSON.stringify({
+      subject: 'Guest token for Webex Calling SDK Sample App',
+      displayName: 'Calling Guest User',
+    }),
+  });
+
+  const data = await response.json();
+
+  return data.accessToken;
+}
+
+async function handleServiceSelect(e) {
+  const value = e.target.value;
+
+  if(value === 'guestCalling') {
+    // Show guest inputs and hide access token input
+    jwtTokenForDestElm.classList.remove('hidden');
+    saveElm.disabled = true;
+
+    try {
+      const guestAccessToken = await getGuestAccessToken();
+      tokenElm.value = guestAccessToken;
+    } catch (error) {
+      if(error.code === 401) {
+        // TODO: Refresh the access token and try again with the new token
```

Comment:
We will handle this part when we work on [SPARK-544995](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-544995)